week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,11 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = group.every(function (name) {
+  return students.find(function (stName){
+    return name === stName
```

Comment:
This is correct!

There is a shorter solution though. Remember that the `.includes()` method will check if the array (`students` in this case) includes a particular value (`name` in this case). Can you try implementing that solution?

Reply:
ohhh yes i was not clear which function returns what but now i revised them all ..... much clear concept now .... Thanks